packages/entrypoint/src/index.ts

```diff
@@ -15,8 +15,9 @@ declare const window: { [x: string]: any };
 
 // We would override __webpack_public_path__ only if __WPACKIO__ webpack plugin is set
 if (__WPACKIO__) {
+	const path = (__WPACKIO__.appName + __WPACKIO__.outputPath)
+		.toLowerCase()
```

Comment:
Any particular reason why you've opted for `.toLowerCase`? Is it because of `sanitize_key` from WordPress API?

Reply:
Yes, just because of it. But I assume that it can break backward compatibility for those who generating this variable manually. What do you think?

Reply:
Removed this. I think better not to break backward compatibility

Reply:
I think I will merge this and do some testing...